test/tests/ver-1d/ver-1d-trapping.i

```diff
@@ -132,6 +142,17 @@ temperature = 1000
     value = outflux
     scaling_factor = ${cl}
   []
+  [nonlin_it]
+    type = NumNonlinearIterations
+  []
+  [dt]
+    type = TimestepSize
+  []
```

Comment:
```suggestion
```
I don't think we need these. They are good for debugging, but really not needed for a test.